plotly_express/_doc.py

```diff
@@ -91,6 +91,10 @@
         "Ignored if `error_z` is `None`.",
     ],
     color=[colref, "Values from this column are used to assign color to marks."],
+    opacity=[
+        "Sets the opacity of the markers in scatter plots such as `scatter()`",
```

Comment:
I'll tweak this wording a bit. To match the rest it should be something like `(number between 0 and 1) Sets the opacity of markers.` (no need to enumerate the function types as this shows up in the individual doc strings of those functions.)

Reply:
I've adjusted to keep it more inline, feel free to change as necessary!